app/assets/javascripts/char-count.js

```diff
@@ -166,7 +172,10 @@
     // Check for existing info count message
     var countMessage = document.getElementById(elementId + '-info')
     // If there is no existing info count message we add one right after the field
-    if (elementId && !countMessage) {
```

Comment:
@Muhammad-Ali-Ahmad I'm rereading this now and I think earlier I missed the fact that there was already a guard on this one. My apologies, I did a search for "insertAdjacentHTML" and didn't look closely enough at the context. The original code on this part should be ok to revert to